.coderabbit.yaml

```
@@ -3,28 +1,33 @@
   guidelines:
-    - Verify Manifest V3 compliance
-    - Check permissions scope for "scripting", "storage", and "activeTab"
-    - Review content script injection patterns
-    - Validate cross-origin XHR requests in background scripts
-    - Ensure proper error handling in API calls
-    - Check GDPR compliance for data collection
-    - Verify webpack build optimization
-
-feedback: true
-auto_approve: true
+  - "Verify Manifest V3 compatibility and compliance"
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)